### ISSUE TITLE
undefined observersFor/LOG_BINDINGS

### ISSUE THREAD

**Issue by jjaffeux** [emberjs/ember.js#18929]
Hi,

I was reading at the debugging documentation https://guides.emberjs.com/v3.12.0/configuring-ember/debugging/#toc_observers--binding and tried to uses both solutions described here, but the LOG_BINDINGS does nothing visible, and the other one gets me an undefined function.

Searching for these in Ember code base, I found no results.

Is this still existing/usable ? Am I missing something? Should the documentation be updated?

Thanks.

**Comment by CvX** [comment]
`LOG_BINDINGS` has been removed together with the `Ember.Binding` support in https://github.com/emberjs/ember.js/commit/f6d6ee634a1fdadf256ff0dde0a115fa1a04d002 and https://github.com/emberjs/ember.js/commit/fcec6311c76709166b76fe0d1d8aee8f567d0c07.

As far I can tell there's no `observersFor` replacement, and as for `LOG_BINDINGS` I guess the modern counterpart would be a debug glimmer-vm build? (with some debug flags enabled: [glimmer-vm/packages/-glimmer/local-debug-flags/index.ts](https://github.com/glimmerjs/glimmer-vm/blob/4aea36d6864ac0bf28e10fefa1cb43efb56886f7/packages/%40glimmer/local-debug-flags/index.ts), [glimmer-vm/build/broccoli/strip-glimmer-utilities.js](https://github.com/glimmerjs/glimmer-vm/blob/4aea36d6864ac0bf28e10fefa1cb43efb56886f7/build/broccoli/strip-glimmer-utilities.js))

**Comment by jjaffeux** [comment]
Yes been looking at debug flags, but really it's more like a workaround, than a counterpart. It's much more involved than previous logging behaviour.

I guess you confirm my point that doc should be updated.

**Comment by rwjblue** [comment]
Ya, this should be updated in the guides. 

@emberjs/learning-team-managers - Best path forward here?

**Comment by jenweber** [comment]
It looks like it should be removed all the way back to 3.1, correct? If so, we can definitely do that! We edit older guides versions for broken links and things that are factually/syntactically incorrect. I opened a ticket here: https://github.com/ember-learn/guides-source/issues/1443

**Comment by rwjblue** [comment]
Ya, it just doesn’t _do_ anything any more and we missed it during the 3.0.0 run up.

**Comment by kategengler** [comment]
This has been handled.
